lisa/sut_orchestrator/azure/features.py

```diff
@@ -3263,6 +3264,8 @@ def create_file_share(
             allow_shared_key_access=allow_shared_key_access,
         )
 
+        # The SMB endpoint https://<share>.file.core.windows.net
+        # will auto point to <share>.privatelink.file.core.windows.net
```

Comment:
The added comments is always true, or only when `enable_private_endpoint` is True? If it happens when `enable_private_endpoint` is true, move the code comment to right place.

Reply:
updated comment to ensure clarity on condition and sequence